tests/run/named-tuples.scala

```diff
@@ -0,0 +1,83 @@
+import annotation.experimental
+import language.experimental.namedTuples
+import NamedTuple.dropNames
+
+type Person = (name: String, age: Int)
+val bob = (name = "Bob", age = 33): (name: String, age: Int)
+val person2: (name: String, age: Int) = bob
+
+type Uni = (uni: Double)
+val uni = (uni = 1.0)
+val _: Uni = uni
+
+type AddressInfo = (city: String, zip: Int)
+val addr = (city = "Lausanne", zip = 1003)
+val _: AddressInfo = addr
+
+type CombinedInfo = Tuple.Concat[Person, AddressInfo]
+val bobWithAddr = bob ++ addr
+val _: CombinedInfo = bobWithAddr
+val _: CombinedInfo = bob ++ addr
+
+@main def Test =
+  println(bob)
+  println(bob.age)
+  println(person2.name)
+  println(bobWithAddr)
+  bob match
+    case p @ (name = "Bob", age = _) => println(p.age)
+  bob match
+    case p @ (name = "Peter", age = _) => println(p.age)
```

Comment:
Is it possible to use a variable pattern on the right-hand side of the `=` sign?

```suggestion
    case (name = "Peter", age = age) => println(age)
```

Reply:
yes, the latest tests contain that pattern.